src/PhpGenerator/Method.php

```diff
@@ -392,4 +404,23 @@ public function setNamespace(PhpNamespace $val = NULL)
 		return $this;
 	}
 
+	/**
+	 * @param  string|NULL
+	 * @return self
+	 */
+	public function setReturnType($val)
```

Comment:
Do you want me to change the field type itself and getter too?

Also, `$visibility` (and related) work with `string|NULL` as well - is that intended, or a little inconsistency?